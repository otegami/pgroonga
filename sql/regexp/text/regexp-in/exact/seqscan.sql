CREATE TABLE memos (
  id integer,
  content text
);

INSERT INTO memos VALUES (1, 'PostgreSQL');
INSERT INTO memos VALUES (2, 'Groonga');
INSERT INTO memos VALUES (3, 'PGroonga');

SET enable_seqscan = on;
SET enable_indexscan = off;
SET enable_bitmapscan = off;

SELECT id, content
  FROM memos
 WHERE content &~| ARRAY['\Agroonga\z', '\Apostgresql\z'];

DROP TABLE memos;
