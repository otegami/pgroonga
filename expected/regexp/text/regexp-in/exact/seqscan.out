CREATE TABLE memos (
  id integer,
  content text
);
INSERT INTO memos VALUES (1, 'PostgreSQL');
INSERT INTO memos VALUES (2, 'Groonga');
INSERT INTO memos VALUES (3, 'PGroonga');
SET enable_seqscan = on;
SET enable_indexscan = off;
SET enable_bitmapscan = off;
SELECT id, content
  FROM memos
 WHERE content &~| ARRAY['\Agroonga\z', '\Apostgresql\z'];
 id |  content   
----+------------
  1 | PostgreSQL
  2 | Groonga
(2 rows)

DROP TABLE memos;
